cmake/portaudio.def.in

```diff
@@ -39,6 +39,7 @@ Pa_GetStreamWriteAvailable          @32
 Pa_GetSampleSize                    @33
 Pa_Sleep                            @34
 Pa_GetVersionInfo                   @35
+; add new portable public API functions here. DO NOT CHANGE EXISTING ORDINALS!
```

Comment:
Why is this change in this PR?

Reply:
An accident. I've rebased onto master and force pushed a new version.